lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -330,13 +330,20 @@ <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
                   <b class="message-copy-bold">
                     ${_("Pursue a {cert_name_long} to highlight the knowledge and skills you gain in this course.").format(cert_name_long=cert_name_long)}
                   </b><br>
-                    ${_("It's official. It's easily shareable. It's a proven motivator to complete the course. <br>{link_start}Learn more about the verified {cert_name_long}{link_end}.").format(link_start='<a href="{}" class="verified-info" data-course-key="{}">'.format(marketing_link('WHAT_IS_VERIFIED_CERT'), enrollment.course_id), link_end="</a>", cert_name_long=cert_name_long)}
+                    ${Text(_("It's official. It's easily shareable. "
+                        "It's a proven motivator to complete the course. {line_break}"
+                        "{link_start}Learn more about the verified {cert_name_long}{link_end}.")).format(
+                          line_break=HTML('<br>'),
+                          link_start=HTML('<a href="{}" class="verified-info" data-course-key="{}">'.format(
+                            marketing_link('WHAT_IS_VERIFIED_CERT'),
+                            enrollment.course_id
+                          )), link_end=HTML('</a>'), cert_name_long=cert_name_long)}
```

Comment:
I'd put these last two on their own lines as well. This shouldn't block you if you don't want to do this, but it reads so much nicer, even if the string stays on one line. Same for the rest.